servicegeneral-dbscripts/db_3_30.sql

```diff
@@ -0,0 +1,195 @@
+-- MySQL Script generated by MySQL Workbench
+-- Tue 30 Mar 2021 03:56:56 PM EDT
+-- Model: New Model    Version: 1.0
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
```

Comment:
The script looks good but please push it to the dev branch first by creating a Pull request between your feature branch and the dev branch. 
Then we can have another Pull request from the dev to the main branch. Let's avoid merging code directly to the main branch.
Please note that the main branch will be merged only by the dev branch.